common/stack.go

```diff
@@ -35,6 +37,22 @@ type StackDeleter interface {
 	DeleteStack(stackName string) error
 }
 
+// S3StackDeleter for purging files from S3 buckets
```

Comment:
we shouldn't have resource specific delete functions one the CFN interface.  there are existing interfaces for these entities so the functions should be cohesive with those rather than the stack

Reply:
Okay, I'll look for them.

Reply:
I think I understand what you're getting at, but I don't see pre-existing interfaces for ecr, buckets.